src/zoid/buttons/util.js

```diff
@@ -18,6 +18,30 @@ export function determineFlow(props : ButtonProps) : $Values<typeof BUTTON_FLOW>
     }
 }
 
+export function mightUseQRPay(funding : $Values<typeof FUNDING>) : boolean {
```

Comment:
Can we use the prefix `supports`? We do that in a few other spots in this repo:
```suggestion
export function supportsQRPay(funding : $Values<typeof FUNDING>) : boolean {
```